plugins/plugin-core-dev/src/connection.ts

```diff
@@ -107,5 +120,19 @@ class ClientImpl implements WithAccountId {
 }
 
 export async function connect (handler: (tx: Tx) => void): Promise<WithAccountId> {
-  return await ClientImpl.create(handler)
+  const client = await ClientImpl.create(handler)
+  const ddProcessor = await DerivedDataProcessor.create(client.model, client.hierarchy, newClientOnlyStorage(client))
```

Comment:
It is not required, since it is already in core client.

Reply:
Required... Because in core client we don't save dd transaction in DB

Reply:
And we should not save them

Reply:
We just need notification to update live queries

Reply:
Ok, since it server simulation we need it.